app/templates/partials/calendar/calendar_base.html

```diff
@@ -1,23 +1,24 @@
 {% extends "./partials/base.html" %}
 {% block head %}
-  {{ super() }}
+{{ super() }}
 
-  <!-- CSS -->
-  <link href="{{ url_for('static', path='/grid_style.css') }}" rel="stylesheet" type="text/css">
+<!-- CSS -->
+<link href="{{ url_for('static', path='/grid_style.css') }}" rel="stylesheet" type="text/css">
 
-  <!-- Scripts -->
-  <script src="{{ url_for('static', path='/js/grid_scripts.js') }}"></script>
+<!-- Scripts -->
+<script src="{{ url_for('static', path='/js/grid_scripts.js') }}"></script>
```

Comment:
The JS should be before the `</body>`

Reply:
Maybe will be better to keep it in the head and set a async \ defer to it ? 
we talked about it in the team and it seemed to have a lot of supporters. 
[Look here.](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup)

I'd be happy to know what do you think about this method.